packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -43,6 +43,15 @@ export function createKusama (t: TFunction): EndpointOption {
       /// (3) parachains with id, see Rococo (info here maps to the actual "named icon")
       //
       // NOTE: Added alphabetical based on chain name
+      {
+        info: 'altair',
+        isUnreachable: true,
+        paraId: 2021,
+        text: t('rpc.kusama.altair', 'Altair', { ns: 'apps-config' }),
+        providers: {
+          Centrifuge: 'wss://fullnode.altair.centrifuge.io/ws'
```

Comment:
the port associated with the ingress service already forwards to the ws one, so no need for the trailing `/ws`